client/my-sites/comments/comment-replies-list/style.scss

```diff
@@ -1,3 +1,8 @@
 .comment-replies-list {
 	margin-left: 56px;
 }
+
+// Show the box shadow for the first two 2 replies
+.comment-replies-list:nth-child(n+2) {
```

Comment:
What's this rule trying to do?

This will change the margin if there are more than two `<div class="comment-replies-list"></div>` elements side-by-side, but as far as I can tell that never happens.